client/src/signin/Register.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from "react";
+import axios from "axios";
+
+class Register extends Component {
+  state = {
+    username: "",
+    department: "",
+    password: "",
+    error: ""
+  };
+
+  render() {
+    return (
+      <div class="form-wrapper">
+        <h2>Login</h2>
+        <form onSubmit={this.submitHandler}>
+          <input
+            name="username"
+            id="username"
+            value={this.state.username}
+            placeholder="Username"
+            type="text"
+            required
+            onChange={this.changeHandler}
+          />
+          <input
+            name="department"
+            id="department"
+            value={this.state.department}
+            placeholder="Department"
+            type="text"
+            required
+            onChange={this.changeHandler}
+          />
+          <input
+            name="password"
+            id="password"
+            value={this.state.password}
+            placeholder="Password"
+            type="password"
+            required
+            onChange={this.changeHandler}
+          />
```

Comment:
These inputs seem repetitive; do you think you could make a component for them? Or, for the sake of an example, a method:
```javascript
const makeInput = (name, type = 'text') => (
  <input
    name={name}
    id={name}
    placeholder={name[0].toUpperCase() + name.slice(1)}
    type={type}
    value={this.state[name]}
    onChange={this.changeHandler}
  />
);
```
and the usage:
```javascript
<form onSubmit={this.submitHandler}>
  {this.makeInput('username')}
  {this.makeInput('department')}
  {this.makeInput('password', 'password')}
  <button type="submit">Register</button>
</form>
```